### ISSUE TITLE
Add `include_path` option to get_stripped_url

### ISSUE THREAD

**Issue by birdsarah** [mozilla/domain_utils#10]
Continue increasing utility of get_stripped_url by adding a boolean `include_path`. Default will be True for current behavior to continue.

```python
def test_get_stripped_url_when_no_path_requested():
    url = 'my.example.com/path/to/webapp.htm?aced=1'
    result = du.get_stripped_url(url, include_path=False)
    assert result == 'my.example.com'
```

```diff
diff --git a/domain_utils/domain_utils.py b/domain_utils/domain_utils.py
index de3436a..b10d0ba 100644
--- a/domain_utils/domain_utils.py
+++ b/domain_utils/domain_utils.py

@@ -122,10 +126,10 @@ def hostname_subparts(url, include_ps=False, **kwargs):
     return subparts
 
 
-def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True):
+def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True, include_path=True):
     """
     Returns a url stripped to just the beginning and end, or more formally,
-    ``(scheme)?+netloc+path``.
+    ``(scheme)?+(netloc|hostname)+(path)?``.
     For example ``https://my.domain.net/a/path/to/a/file.html#anchor?a=1``
     becomes ``my.domain.net/a/path/to/a/file.html``
     URL parsing is done using std lib
@@ -156,7 +160,10 @@ def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True):
         that a port is included, for example, if it was in the path.
         Default is ``True``.
     :type use_netloc: bool, optional
-    :return: Returns a url stripped to (scheme)?+netloc+path.
+    :param include_path: If ``True`` return value will include the 
+        url path. Default is ``True``.
+    :type include_path: bool, optional
+    :return: Returns a url stripped to (scheme)?+(netloc|hostname)+(path)?.
         Returns empty string if appropriate.
     :rtype: str
     """
@@ -179,7 +186,7 @@ def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True):
     purl = urlparse(url)
     scheme_out = ''
     loc_out = ''
-    path_out = purl.path
+    path_out = ''
 
     if scheme is True:
         if _scheme in ['http', 'https']:
@@ -193,6 +200,9 @@ def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True):
     else:
         loc_out = purl.hostname
 
+    if include_path is True:
+        path_out = purl.path
+
     return '{scheme_out}{loc_out}{path_out}'.format(
         scheme_out=scheme_out,
         loc_out=loc_out,
```

**Comment by birdsarah** [comment]
The argument should just be `path` in the same way that we just have the argument `scheme`. To prevent epic merge drama, suggest wait on implementing till #22 is merged.

**Comment by birdsarah** [comment]
Fixed by 6e9669b14272071f52f2ecf2a2c453c5d52dede8 (meant to do it as a PR, inadvertently committed to master).
